x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/profile/ProfileService.java

```diff
@@ -128,7 +135,7 @@ public void activateProfile(Authentication authentication, ActionListener<Profil
 
         getVersionedDocument(subject, ActionListener.wrap(versionedDocument -> {
```

Comment:
Should we rename this to "searchVersionedDocumentForSubject"? Particularly I would suggest that the method name contain "search" instead of "get".

Reply:
Good point. I updated as you suggested.